### PR TITLE
Added "Sharding" section

### PR DESCRIPTION
**Pull request by McFranko**
Sharding is important for scaling and throughput, but could be a lot to promise for the first release. It could potentially be included as a later upgrade instead, or it could be included from the beginning.

### CODE REVIEW

File: whitepaper.tex [hunk 1]
```diff
@@ -267,6 +267,28 @@ \section{Anti-Spam Measures}
 Instead of being economically expensive to execute a DOS attack, PADLOCK makes
 it very computationally expensive.
 
+
+\section{Sharding}
+One major disadvantage of a blockchain is how full nodes must store the entire
+chain. This becomes inefficient, as you have many copies of the same data, when
```
Comment (lindenstark) [inline whitepaper.tex:273]: "you" implies reader is a blockchain

Comment (lindenstark) [inline whitepaper.tex:273]: Extra comma after "data"

File: whitepaper.tex [hunk 2]
```diff
@@ -267,6 +267,28 @@ \section{Anti-Spam Measures}
...
+these nodes could instead store different data, taking advantage of the full
+storage and computational power of the network. In PADLOCK this will be achieved
+with horizontal sharding. PADLOCK will consist of many chains. A node will only
+store the data of one chain. When a client wants to find something such as the
+merkle-proof of an entry, they will have to request it from one node of each
```
Comment (lindenstark) [inline whitepaper.tex:278]: "merkle" should be capitalized

File: whitepaper.tex [hunk 3]
```diff
@@ -267,6 +267,28 @@ \section{Anti-Spam Measures}
...
+chain. The mempool will be shared between all nodes. Miners can choose to mine
+on any chain. The hash power going into each chain will likely be balanced, as a
+chain with a lower hash rate will quickly be picked up by miners taking
+advantage of the potential profits.
+
+Miners could potentially include the same entries in multiple blocks across
+chains. This would result in an inefficient use of space. So, there is a
```
Comment (lindenstark) [inline whitepaper.tex:285]: "in [an] inefficient" "an" is not needed

**Review by lindenstark — CHANGES_REQUESTED** [review]
There's a few grammatical mistakes.
Should we look for grammarly as a CI service, it'd make contributing scale a lot smoother?

**Comment by McFranko** [comment]
Grammarly would only introduce spelling fixes, unless I pay for the premium version, which I'm not doing. I just didn't proof read this at all.

**Comment by McFranko** [comment]
I don't think I will merge this anyways because since writing this I've realized the insecurity of a side-chain scaling solution. Everytime you add another chain you lower the hash power it takes to attack the network. (1 chain = 50%, 2 chains = 25%, 4 chains = 12.5%, etc).

**Pull request closed without merging**